compiler/test_evaluation_scripts.sh

```diff
@@ -23,6 +23,7 @@ n_inputs=(
 configurations=(
     ""
     "--r_split"
+    "--dgsh_tee"
```

Comment:
Maybe we want to have a combination of r_split and dgsh tee (since that is the optimal one that we really care about anyway.

Reply:
Added it!